src/components/vehicleDataGroup/VehicleDataGroupField.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC, ReactNode } from 'react';
+import { Box, chakra, ListItem, useMultiStyleConfig } from '@chakra-ui/react';
+
+interface VehicleGroupDataFieldProps {
+  label: ReactNode;
```

Comment:
I'm wondering the type of this prop. Shouldn't be a string?

Reply:
> I'm wondering the type of this prop. Shouldn't be a string?

I have in mind option to pass React Comp as label and value. If we need to add Icon + text as label for instance.

Reply:
True, true!
I didn't checked the design.
👍

Reply:
but we don't have that in the design, right? For me this should also be a string and not a ReactNode

Reply:
I was checking the basic details but it should be the data group and data field. Therefore, I will go for a string as my first assumption